### PR TITLE
fix SynchronousSceneManager JavaDocs

### PR DESCRIPTION
**Pull request by ShirleyNekoDev**

### CODE REVIEW

File: chunky/src/java/se/llbit/chunky/renderer/scene/SynchronousSceneManager.java [hunk 1]
```diff
@@ -44,22 +44,24 @@
...
+   * a reset confirm dialog will be shown before applying any further
+   * non-transitory changes.
    *
-   * Multiple threads can try to read/write the mutable scene concurrently,
+   * <p>Multiple threads can try to read/write the mutable scene concurrently,
    * so multiple accesses are serialized by the intrinsic lock of the Scene
    * class.
    *
-   * NB: lock ordering for scene and storedScene is always scene->storedScene!
+   * <p><strong>NB: lock ordering for scene and storedScene is always scene->storedScene!
```
Comment (leMaik) [inline chunky/src/java/se/llbit/chunky/renderer/scene/SynchronousSceneManager.java:57]: What does NB mean?

**Review by leMaik — APPROVED** [review]

File: chunky/src/java/se/llbit/chunky/renderer/scene/SynchronousSceneManager.java (hunk 1 above)
Comment (ShirleyNekoDev) [reply chunky/src/java/se/llbit/chunky/renderer/scene/SynchronousSceneManager.java:57]: idk

Comment (leMaik) [reply chunky/src/java/se/llbit/chunky/renderer/scene/SynchronousSceneManager.java:57]: I was today years old when i leaned... https://www.britannica.com/dictionary/NB#:~:text=%E2%97%8A%20The%20abbreviation%20NB%20comes,which%20means%20%E2%80%9Cmark%20well.%E2%80%9D

Comment (leMaik) [inline chunky/src/java/se/llbit/chunky/renderer/scene/SynchronousSceneManager.java:57]:
```suggestion
   * <p><strong>Important: lock ordering for scene and storedScene is always scene->storedScene!
```

**Comment by NotStirred** [comment]
this might be a good time to fix the incorrect lock ordering in SynchronousSceneManager#saveSceneAs

**Pull request merged**